### DISCUSSION TITLE
Easy to use tool to edit subway surfers data

### DISCUSSION THREAD

**Discussion by le0o0oo** [Noobgamer0111/SubwaySurfersDecrypted#13] [Ideas]
I made this online tool where you paste the encrypted json file content and it will allow you to modify it, without using any external old files.

Link: https://subway-surfers-ez-gui.vercel.app/
Github: https://github.com/Farfa7886/subwaySurfers-ezGUI/

The decryption key has been found by @moddedmcplayer

**Comment by ghost** [comment]


**Reply by le0o0oo** [reply]
What do you mean? Can you post your input and the output? Contact me on discord if needed since I will reply faster there
